docs/x509/reference.rst

```diff
@@ -610,18 +704,35 @@ X.509 Revoked Certificate Object
 
         An integer representing the serial number of the revoked certificate.
 
+        .. doctest::
+
+            >>> crl.revoked_certificates[0].serial_number
```

Comment:
For this doctest (and others using `RevokedCertificate`) I'd prefer that we assigned `revoked_certificate = crl.revoked_certificates[0]` so that this would be `revoked_certificate.serial_number`.